src/main/scala/firrtl/FileUtils.scala

```diff
@@ -159,4 +184,13 @@ object FileUtils {
     inputStream.close()
     text
   }
+
+  /** Get os.Path from String
+    * @param pathName an absolute or relative path string
+    */
+  def getPath(pathName: String): os.Path = os.FilePath(pathName) match {
+    case path: os.Path    => path
+    case sub:  os.SubPath => os.pwd / sub
+    case rel:  os.RelPath => os.pwd / rel
+  }
```

Comment:
I'm a bit late here, but it seems this is enshrining use of absolute paths which I understand is convenient but is actually generally not a great thing to do since it makes runs not relocatable (eg. if I emit files with absolute paths in them, then the one in my workspace is different than the one in your work space).

To be fair, I'm not sure if the existing APIs are already very "absolutely path centric" but it's generally something we should move away from rather than encouraging.